ichub-backend/controllers/fastapi/routers/authentication/auth_api.py

```diff
@@ -0,0 +1,69 @@
+#################################################################################
+# Eclipse Tractus-X - Industry Core Hub Backend
+#
+# Copyright (c) 2025 LKS Next
+# Copyright (c) 2025 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied. See the
+# License for the specific language govern in permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+#################################################################################
+
+from tractusx_sdk.dataspace.managers import AuthManager, OAuth2Manager
+from managers.config.config_manager import ConfigManager
+from fastapi.security import APIKeyHeader, HTTPBearer, HTTPAuthorizationCredentials
+from fastapi import HTTPException, Request, status, Depends
+
+auth_manager: AuthManager | OAuth2Manager = None
+
+if not ConfigManager.get_config("authorization.keycloak.enabled"):
+    auth_manager = AuthManager(
+        api_key_header=ConfigManager.get_config("authorization.api_key.key"),
+        configured_api_key=ConfigManager.get_config("authorization.api_key.value"),
+        auth_enabled=ConfigManager.get_config("authorization.enabled")
```

Comment:
```suggestion
        auth_enabled=ConfigManager.get_config("authorization.enabled")
```

if the authorization is disabled. we should also not do the keycloak login. We shall just ignore and give back the "true" for the authorization.

Reply:
Done [c982961](https://github.com/eclipse-tractusx/industry-core-hub/pull/326/commits/c982961bc7996c3d5bf204af4f90e686634f56ee)